### PR TITLE
Add missing help texts

### PR DESCRIPTION
**Pull request by vxsx**
this is a wip PR to add all the help texts so autocompletion could just use `lancet _arguments command` for everything
@GaretJax do you want me to put the zsh completion file somewhere in the project as well or should I still keep it [here](https://github.com/vxsx/dotfiles/blob/master/zsh/lancet.zsh)?

### CODE REVIEW

File: lancet/commands/configuration.py [hunk 1]
```diff
@@ -57,7 +57,8 @@ def setup(ctx, force):
 
 
 @click.command()
-@click.option('-f', '--force/--no-force', default=False)
+@click.option('-f', '--force/--no-force', default=False,
+              help=('Init even if .lancet already exists.'))
```
Comment (GaretJax) [inline lancet/commands/configuration.py]: Can we remove the parenthesis here pls?

**Comment by GaretJax** [comment]
@vxsx we might merge it somewhen, but I want to setup a command to load the whole file as I did for `_setup_helper`.

File: lancet/commands/configuration.py (hunk 1 above)
Comment (vxsx) [inline lancet/commands/configuration.py]: yeah sure

**Comment by vxsx** [comment]
@GaretJax I was poking around in the code and it seems like some things would be a bit hard to do complete automatically (well, I don't have sufficient knowledge of zsh autocompletion possibilities/syntax ofc). Also calling `lancet _arguments $command` is a quite slow compared to hardcoded version, so maybe we could inline these on `lancet _autocomplete` call or something. Not sure how to correctly update this autocompletion file on lancet update afterwards, though, but that's something else.

Help texts are ready, please take a look, if you have any corrections or anything. I'd like to squash before/if you're gonna merge.

**Comment by GaretJax** [comment]
While generating the whole file with inlined commands may be a solution, we could also look into ZSH autocompletion caching. 
(http://zsh.sourceforge.net/Doc/Release/Completion-System.html search for `use-cache`)

**Comment by vxsx** [comment]
@GaretJax can we merge this one? zsh completion improvements will follow in separate PR

**Comment by GaretJax** [comment]
:+1:

**Pull request merged**